app/content/team/index.js

```diff
@@ -252,48 +252,62 @@ function Index() {
             <Header
               title={"Team"}
               postChildren={
-                currentUserInfo.role === "owner" ? (
-                  <Menu
-                    visible={menuVisible}
-                    onDismiss={() => {
-                      setMenuVisible(false);
-                    }}
-                    anchor={
-                      <Appbar.Action
-                        icon="dots-horizontal-circle-outline"
-                        onPress={() => {
-                          setMenuVisible(true);
-                        }}
-                        color={themeColors.accent}
-                      />
-                    }
-                    statusBarHeight={45}
-                    anchorPosition="bottom"
-                    contentStyle={{
-                      backgroundColor: themeColors.background,
-                    }}
-                  >
-                    <Menu.Item
-                      leadingIcon="pencil-outline"
+                <>
+                  {currentUserInfo.role === "coach" ||
+                  currentUserInfo.role === "owner" ? (
```

Comment:
In the case where we don't add anymore roles, could we do this?
```suggestion
                  {currentUserInfo.role !== "player" ? (
```
Might be better to leave it like this though. Just a thought

Reply:
I'd say let's keep it explicit